src/components/ReactQueryProvider/index.tsx

```diff
@@ -0,0 +1,33 @@
+"use client";
+
+import React, { useState } from "react";
+import { QueryClientProvider, QueryClient } from "@tanstack/react-query";
+import { ReactQueryDevtools } from "@tanstack/react-query-devtools";
+
+interface Props {
+  children: React.ReactNode;
+}
+
+function ReactQueryProvider({ children }: Props) {

```

Comment:
[ P3 ] 저희 컨벤션에는 요게 좀 더 좋지 않을까 의견드려봅니다! 🙋‍♂️

```suggestion
interface ReactQueryProviderProps {
  children: React.ReactNode;
}

const ReactQueryProvider = ({ children }: ReactQueryProviderProps) => {
```

Reply:
좋은 의견이십니다!! 👍👍
급하게 처리하느라 생각 없이 한 것 같은데.. 수정하도록 하겠습니다!! (아마 해당 코드는 원주님 코드로 대체될 것 같네요 ㅎㅎ)